mono/mini/interp/interp.c

```diff
@@ -2631,6 +2631,31 @@ interp_entry_from_trampoline (gpointer ccontext_untyped, gpointer rmethod_untype
 }
 #endif /* MONO_ARCH_HAVE_INTERP_ENTRY_TRAMPOLINE */
 
+/* For debug use */
+void
+mono_interp_print_code (InterpMethod *imethod)
+{
+	MonoJitInfo *jinfo = imethod->jinfo;
+	const guint16 *p, *start, *end;
+
+	if (!jinfo)
+		return;
+
+	char *name = mono_method_full_name (imethod->method, 1);
+	g_print ("Method : %s\n", name);
+	g_free (name);
+
+	p = start = (guint16*) jinfo->code_start;
+	end = p + jinfo->code_size;
+
+	while (p < end) {
+		char *ins = mono_interp_dis_mintop (start, p);
+		g_print ("%s\n", ins);
+		g_free (ins);
+		p = mono_interp_dis_mintop_len (p);
```

Comment:
slightly duplicated code with `dump_mint_code`, but I guess it doesn't matter 🙂

Reply:
@lewurm Added a test case, which failed on interp only. With the jit, even though we have a MonoMethod with an invalid signature in the wrapper data, it doesn't seem to be used anywhere in simple code.